index.html

```diff
@@ -912,16 +912,16 @@ <h5>Our Headquarters</h5>
                 <p>Intrinsicly matrix high standards in niches whereas intermandated niche markets. Objectively harness competitive resources.</p>
                 <ul class="list-unstyled social-list mb-0">
                   <li class="list-inline-item">
-                    <a href="#" class="rounded"><span class="ti-facebook white-bg color-2 shadow rounded-circle"></span></a>
+                    <a href="#" class="rounded" aria-label="Visit us on Facebook" title="Facebook (External Link)" rel=”noopener noreferrer”><span class="ti-facebook white-bg color-2 shadow rounded-circle"></span></a>
                   </li>
                   <li class="list-inline-item">
-                    <a href="#" class="rounded"><span class="ti-twitter white-bg color-2 shadow rounded-circle"></span></a>
+                    <a href="#" class="rounded" aria-label="Visit us on Twitter" title="Twitter (External Link)" rel=”noopener noreferrer”><span class="ti-twitter white-bg color-2 shadow rounded-circle"></span></a>
                   </li>
                   <li class="list-inline-item">
-                    <a href="#" class="rounded"><span class="ti-linkedin white-bg color-2 shadow rounded-circle"></span></a>
+                    <a href="#" class="rounded" aria-label="Visit us on Linkedin" title="Linkedin (External Link)" rel=”noopener noreferrer”><span class="ti-linkedin white-bg color-2 shadow rounded-circle"></span></a>
                   </li>
                   <li class="list-inline-item">
-                    <a href="#" class="rounded"><span class="ti-dribbble white-bg color-2 shadow rounded-circle"></span></a>
+                    <a href="#" class="rounded" aria-label="Visit us on Dribble" title="Dribble (External Link)" rel=”noopener noreferrer”><span class="ti-dribbble white-bg color-2 shadow rounded-circle"></span></a>
```

Comment:
You just messed the whole thing here! Why didn't you check this before pushing? 😞  @0xabdulkhalid

Reply:
Bro, I can't get what you're talking about ?

Neenga konjam theliva sollunga Anna, yenakkum onnum puriyalai 🥲

Reply:
Yeah i got that, `rel=”noopener noreferrer”`.

This part right ?

Reply:
Adhey dhaan, the quotes are smart quotes not the normal double quotes! 😞

Reply:
Sorry for the inconvenience Anna, next time i'll verify the changes before opening the PR.

Reply:
Thanks da. 😊